pcdsdevices/pim.py

```diff
@@ -1,23 +1,21 @@
 """
-Profile Intensity Monitor Classes
-
-This module contains all the classes relating to the profile intensity monitor
-classes at the user level. A PIM will usually have at least a motor to control
-yag position and a camera to view the yag.
+Module for the `PIM` profile intensity monitor classes
 """
 import logging
 
-from ophyd import FormattedComponent as FCpt
+from ophyd.device import Device, Component as Cpt, FormattedComponent as FCpt
+from ophyd.signal import EpicsSignal
 
+from .epics_motor import IMS
 from .areadetector.detectors import PCDSAreaDetector
 from .inout import InOutRecordPositioner
 
 logger = logging.getLogger(__name__)
 
 
-class PIMMotor(InOutRecordPositioner):
+class PIM_Y(InOutRecordPositioner):
```

Comment:
Here too, typically the underscore isn't used in class names by python convention
https://www.python.org/dev/peps/pep-0008/#class-names

Reply:
Now been changed to PIMY